#controller buttons
bool a
bool b
bool x
bool y
bool lb
bool rb
bool back
bool start
bool l_thumb
bool r_thumb

#controller axes
int8 x_l_thumb
int8 y_l_thumb
int8 x_r_thumb
int8 y_r_thumb
int8 l_trig
int8 r_trig
